drivers/sensor/lis2mdl/lis2mdl.c

```diff
@@ -450,7 +450,7 @@ static int lis2mdl_set_power_state(struct lis2mdl_data *lis2mdl,
 		const struct lis2mdl_config *const config,
 		enum pm_device_state new_state)
 {
-	const struct lis2mdl_config *cfg = dev->config;
+	const struct lis2mdl_config *cfg = config;
```

Comment:
shouldn't the function just use `config`?

Reply:
correct!

Reply:
done